recipes-kernel/linux/linux-qoriq_4.1.bbappend

```diff
@@ -149,6 +150,7 @@ SRC_URI_append_t700 += "file://${MACHINE}/patches/0002-4.1-Chage-to-fit-T700-NOR
                         file://${MACHINE}/patches/0062-Modify-pwm-function-for-T700.patch                          \
                         file://${MACHINE}/patches/0063-Modify-FAN_REG_CONT-index.patch                             \
                         file://${MACHINE}/patches/0064-ACCTONCIS-82-reduce-the-i2c-access-times-during-upda.patch  \
+                        file://${MACHINE}/patches/0065-Modify-I2C-MDIO-function.-It-return-error-if-bus-acc.patch  \
```

Comment:
Why this pull request affect T700?
Could you remove it?

Reply:
This patch return error to application when I2C/MDIO access fail. T700 DHAL driver need it to check I2C error in PM function.

I am confuse about PGM release. I though that last PGM had include 0065 patch because PM function need it, but this patch don't approve on git.

Reply:
https://github.com/FujitsuNetworkCommunications/meta-fss2-ppc/pull/98/files
00065 patch still open state.
Because if we apply it, we see many flt issue.
After apply I2C driver changing from HUDS, we can merge 00065.
So, please remove 00065 patch from this pull request.